internal/store/organizations.go

```diff
@@ -55,13 +56,15 @@ func (s *Store) Organizations(opts *atlas.OrganizationsListOptions) (*atlas.Orga
 }
 
 // Organization encapsulate the logic to manage different cloud providers.
-func (s *Store) Organization(id string) (*atlas.Organization, error) {
+func (s *Store) Organization(id string) (*v1alpha.ApiOrganizationView, error) {
 	switch s.service {
 	case config.CloudService, config.CloudGovService:
-		result, _, err := s.client.(*atlas.Client).Organizations.Get(s.ctx, id)
+		result, res, err := s.clientv2.OrganizationsApi.GetOrganization(s.ctx, id).Execute()
+		defer res.Body.Close()
```

Comment:
doesn't the generated code handle this? this feels like a big overside on the generated client

Reply:
I want to create separate jira for this specifically.
Body is closed already. Client does handle it. 
It is well known bug in the golangcilint to flag those as not closed when response is returned from method. I will replace this statement with golang cilint exception and link to jira

Reply:
Created Jira: CLOUDP-161495